daml-lf/transaction/src/test/scala/com/digitalasset/daml/lf/transaction/VersionTimelineSpec.scala

```diff
@@ -74,7 +73,7 @@ class VersionTimelineSpec
       inside(inAscendingOrder.last) {
         case Both(
             Both(ValueVersion("dev"), TransactionVersion("dev")),
-            LanguageVersion(LanguageVersion.Major.V1, Dev)) =>
+            LanguageVersion(LanguageVersion.Major.V1, LanguageVersion.Minor("dev"))) =>
```

Comment:
What do you think about adding a constant `LanguageVersion.MinorDev` (that evaluates to `LanguageVersion.Minor("dev")` instead of repeating the somewhat magic constant "dev" in multiple places?

Reply:
I was thinking to give constance for all the versions : `v1_6`, `v1_7`, ...,  `v1_dev` in a separate PR.